packages/cli/test/integration/shared/test-db.ts

```diff
@@ -87,9 +87,20 @@ const repositories = [
  */
 export async function truncate(names: Array<(typeof repositories)[number]>) {
 	for (const name of names) {
-		const RepositoryClass: Class<Repository<object>> =
+		let RepositoryClass: Class<Repository<object>>;
+
+		try {
+			// eslint-disable-next-line n8n-local-rules/no-dynamic-import-template
+			RepositoryClass = (await import(`@/databases/repositories/${kebabCase(name)}.repository`))[
+				`${name}Repository`
+			];
```

Comment:
"Disallow non-relative imports in template string argument to `await import()`, because `tsc-alias` as of 1.8.7 is unable to resolve aliased paths in this scenario." ← Why doesn't this apply here and below? Is the lint rule stale or missing an exception?

Reply:
Not sure, tbh. This import seems to work fine.
Also the related issue in `tsc-alias` was closed as completed without any comments https://github.com/justkey007/tsc-alias/issues/156